TasksFromYurijL/src/SimpleTasks/NumberCheck.java

```diff
@@ -0,0 +1,16 @@
+package SimpleTasks;
+
+public class NumberCheck {
+	/**
+	 *  @return Check if the number is odd
+	*/
+	public boolean isOddNumber(int number) {
+		return (number&1)==1;

```

Comment:
you do not need braces here, please see operations precedence https://docs.oracle.com/javase/tutorial/java/nutsandbolts/operators.html